onnxruntime/core/session/inference_session.h

```diff
@@ -469,6 +470,9 @@ class InferenceSession {
   // Threadpool for this session
```

Comment:
>// Threadpool for this session [](start = 2, length = 30)

This comment is no longer sufficient. Need elaborate explanation of what used when.